featureflags/client.py

```diff
@@ -127,7 +127,13 @@ def authenticate(self):
                 'cluster': self._cluster
             }
         )
-        self._client.with_headers({"User-Agent": "PythonSDK/" + VERSION})
+        # Additional headers used to track usage
+        additional_headers = {
+            "User-Agent": "PythonSDK/" + VERSION,
+            "Harness-SDK-Info": f'Python {VERSION} Server',
+            "Harness-EnvironmentID": self._environment_id,
+            "Harness-AccountID": decoded["accountID"]}
+        self._client = self._client.with_headers(additional_headers)
```

Comment:
The return value of this wasn't actually being used previously, so any custom headers would not have been set.